geom/geom/inc/bvh/v2/bvh.h

```diff
@@ -148,8 +148,11 @@ void Bvh<Node>::traverse_top_down(Index start, Stack& stack, LeafFn&& leaf_fn, I
                 top = near_index;
             } else if (hit_right) {
                 top = right.index;
-            }
-            else [[unlikely]] {
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wattributes"
+            // Prevent gcc warnings if the compiler is not c++20 compliant
+            } else [[unlikely]] {
+#pragma GCC diagnostic pop
```

Comment:
To prevent warnings on non-gcc compilers:
```suggestion
#if defined(__GNUC__)
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wattributes"
            // Prevent gcc warnings if the compiler is not c++20 compliant
            } else [[unlikely]] {
#pragma GCC diagnostic pop
#endif
```
or you can also use `#if !defined(_MSC_VER)`, but I don't know for other compilers...